src/event.rs

```diff
@@ -447,15 +447,15 @@ pub enum WindowEvent {
         button: MouseButton,
     },
 
-    /// Touchpad magnification event with two-finger pinch gesture.
+    /// Two-finger pinch gesture, often use for magnification.
     ///
     /// Positive delta values indicate magnification (zooming in) and
     /// negative delta values indicate shrinking (zooming out).
     ///
     /// ## Platform-specific
     ///
-    /// - Only available on **macOS**.
-    TouchpadMagnify {
+    /// - Only available on **macOS** and **iOS**.
+    PinchGesture {
```

Comment:
Would it make sense to add this?
```
#[deprecated(note="use")]
type TypeAlias TouchpadMagnify = PinchGesture;
```

Then macos targets can still compile with warnings?

Reply:
that's not possible on enum variants